pyrightconfig.json

```diff
@@ -0,0 +1,95 @@
+{
+    "typeshedPath": ".",
+    "include": [
+        "stdlib",
+        "stubs"
+    ],
+    "exclude": [
+        "**/@python2",
+        "stdlib/crypto/__init__.pyi",
```

Comment:
This file doesn't seem to exist, now that I look.

Reply:
Yeah, you're right. You can delete that line or leave it in and we'll delete it the next time we update it. It was left over from an earlier version of this config.

Reply:
Deleted it now.